cfgmgr/intfmgr.cpp

```diff
@@ -115,6 +144,16 @@ void IntfMgr::setIntfIp(const string &alias, const string &opCmd,
     int ret = swss::exec(cmd.str(), res);
     if (ret)
     {
+        /*
+            ipv6 setting on the iface might still haven't been enabled by SDK.
+            Retry adding Ipv6 Prefix after enabling the setting
+        */
+        if (retryV6 && enableIpv6Flag(alias))
```

Comment:
I don't think you need a flag because you can check whether opCmd is "add" & ipPrefix.isV6() and enable ipv6 on interface

Reply:
From the meeting with MSFT, they've suggested in trying setting the IP prefix first and then set the disable_ipv6 flag to zero if it fails, since this is relatively a rare condition

Reply:
I see